BreakpadSupport/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Aug 14 14:14:35 MSK 2018
+#Wed Oct 30 11:15:20 CST 2019
```

Comment:
I don't think we should commit it

Reply:
At least not part of the demo app update

Reply:
I've changed it, please check